Doc/library/copy.rst

```diff
@@ -27,7 +27,7 @@ Interface summary:
    Return a deep copy of *x*.
 
 
-.. exception:: error
+.. exception:: Error
```

Comment:
Note: error is an alias for Error although Error is present in `__all__` too

https://github.com/python/cpython/blob/2aabc3200bf03d2ec1aa987e1e20db704948111e/Lib/copy.py#L55-L57

Reply:
I was under the impression that with that comment Python 2 only had error and not Error. But the same alias is present in Python 2 too. So I guess it predates something more than 2.7. I agree with the change over documenting Error. I will wait for another core dev opinion. The git blame is as below : 

```
227b1204681 (Fred Drake             2000-08-17 05:06:49 +0000  55) class Error(Exception):
88869f9787c (Tim Peters             2001-01-14 23:36:06 +0000  56)     pass
88869f9787c (Tim Peters             2001-01-14 23:36:06 +0000  57) error = Error   # backward compatibility
```

Reply:
Documenting `Error` seems alright and seems more modern usage to me.  +1 with this change.

Reply:
Given `Error` is exposed via  `__all__` interface, it is accurate to update the documentation to reflect that.